docs/zkapps/o1js/permissions.mdx

```diff
@@ -135,7 +135,7 @@ To better understand how to leverage permissions to make your smart contract mor
 
 Some smart contracts manage state and token, such as the native MINA token. To prevent malicious actors from withdrawing all funds, use permissions to secure them.
 
-Consider the following `UnsecureContract` smart contract. A similar [simple_zkapp_payment.ts](https://github.com/o1-labs/o1js/blob/main/src/examples/zkapps/simple_zkapp_payment.ts) contract is also provided in the `examples` folder:
+Consider the following `UnsecureContract` smart contract. A similar [simple-zkapp-payment.ts](https://github.com/o1-labs/o1js/blob/main/src/examples/zkapps/simple-zkapp-payment.ts) contract is also provided in the `examples` folder:
```

Comment:
Hi @barriebyron,  

The link was not pointing to the correct file in repo. I have updated it.
Can you please help to review ?

Thanks 😃